builtin/logical/aws/iam_policies_test.go

```diff
@@ -207,7 +207,7 @@ func Test_combinePolicyDocuments(t *testing.T) {
 				`{"Version": "2012-10-17", "Statement": [{"Effect": "Allow", "NotAction": "ec2:DescribeAvailabilityZones", "Resource": "*"}]}`,
 			},
 			expectedOutput: `{"Version": "2012-10-17","Statement":[{"Effect": "Allow","NotAction": "ec2:DescribeAvailabilityZones",	"Resource": "*"}]}`,
-			expectedErr:    false,
+			expectedErr: false,
```

Comment:
Hmm, any idea where this came from? Vault main isn't failing the formatting check, so you might want to pull latest from there.

Reply:
This actually happened after I ran `make fmt`. The `fmt` check in CI was failing, and so I ran `make fmt` hoping to resolve it... But looks like the `fmt` check is still failing 🤔

Reply:
Huh, I just pull the latest from your branch here and `make fmt` results in a new diff. I think try `make fmt` again and commit the diff should work.

Reply:
Fixed in latest commit, thanks!